modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -37,15 +36,31 @@ The update should be atomic; that is, both the `containerImage` and `version` fi
 
 . Terminate any workload using the kernel module on the node being upgraded.
 
-. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node. This unloads the kernel module from the node.
+. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node.
+This unloads the kernel module from the node:
++
+[source,terminal]
+----
+oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>- ```
```

Comment:
```suggestion
oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-
```

Reply:
As far as I know the last '-' is not needed.

Reply:
Removed **- ```**

Reply:
@StephenJamesSmith . I have to admit that I was wrong. The proper command needs to have the ending '-'
for the label to be removed. 

`oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-`

Sorry for the misunderstanding.

Reply:
@cdvultur Put the dash back in.